### PR TITLE
Rename `master` to `trunk`

### PR DESCRIPTION
**Pull request by mikeyarce**
Rename the `master` branch to `trunk`.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     name: New tag
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@trunk
```
Comment (jblz) [inline .github/workflows/deploy.yml]: EDIT: Scratch that!

```suggestion
      - uses: actions/checkout@v2
        with:
          ref: trunk
```

...maybe?

https://github.com/actions/checkout/tree/25a956c84d5dd820d28caab9f86b8d183aeeff3d#checkout-a-different-branch

File: .github/workflows/integration-tests.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ jobs:
       fail-fast: false
     steps:
       - name: Checkout code
-        uses: actions/checkout@master
+        uses: actions/checkout@trunk
```
Comment (jblz) [inline .github/workflows/integration-tests.yml:39]:
```suggestion
        uses: actions/checkout@v2
        with:
          ref: trunk
```

trying...

Comment (jblz) [inline .github/workflows/integration-tests.yml]:
```suggestion
        uses: actions/checkout@v2
```

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (jblz) [reply .github/workflows/deploy.yml]: Actually, I think it should just use the ref from the tag automatically if we use `actions/checkout@v2`

```suggestion
      - uses: actions/checkout@v2
```

**Comment by jblz** [comment]
Switched to `actions/checkout@v2`. See: https://github.com/marketplace/actions/checkout

We don't need to specify a branch for this...it checks out the hash relating to the event that triggered the action run.

**Review by jblz — APPROVED** [review]
I'm not sure how to test the action changes & we could probably do without them, but it seems likely to work.

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ jobs:
     name: New tag
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v2
```
Comment (GaryJones) [inline .github/workflows/deploy.yml:11]: Not a big issue, but this `master` was related to the version of the `actions/checkout` code, not the wp-parsely master branch.

**Pull request merged**